app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(
+            self,
+            visitor: dict
+    ) -> str | NotVaccinatedError | OutdatedVaccineError | NotWearingMaskError:

```

Comment:
The exceptions are not part of the return type, so they shouldn't be included in the type hint for the return value.